css/index.css

```diff
@@ -1,3 +1,265 @@
+    color: #b0b0b0;
+}
+
+header  nav a:hover {
+  text-decoration: underline;
+  color:black;
+}
+
+header .logo {
+    margin: 28px 0 0 91px;
+}
+
+/* Call To Action (CTA) Styles */
+
+.cta {
+    display: flex;
+    margin-top: 72px;
+    justify-content: flex-end;
+}
+
+.cta .cta-text {
+    font-size: 72px;
+    text-align: center;
+    letter-spacing: 5px;
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    margin: 0 82px 0 0;
+}
+
+.cta .cta-text button {
+    border: 1px double black;
+    font-size: 16px;
+    background: white;
+    width: 170px;
+    height: 32px;
+    margin-top: 10px;
+    cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+/* Main Content Styles */
+
+.main-content {
+   border-top: 2px solid black; 
+   border-bottom: 2px solid black;
+   margin: 75px 0 35px 0;
+   padding: 35px 0;
+   font-size: 16px;
+   line-height: 18px;
+}
+
+.main-content .top-content {
+    display: flex;
+}
+
+.main-content .middle-img {
+    display: block;
+    margin: 30px auto;
+}
+
+.main-content .bottom-content {
+    display: flex;
+}
+
+.main-content .bottom-content p {
+    padding-right: 20px;
+}
+
+/* Contact Styles */
+
+.contact p {
+    margin: 15px 0;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+}
+
+
+/* tablet*/
+@media (max-width:800px){
+    .container {
+    width: 100%;
+    background: orange;
+    
+  }
+  header {
+      flex-wrap: wrap;
+  }
+    header nav {
+        width: 768px;
+        padding: 0 30px;
+           }
+    header .logo {
+  display: block;
+  order: -1;
+  margin: 30px auto 0;
+}
+        .cta img{
+    display: none;
+       }
+       .cta {
+           justify-content: center;
+               }
+               .cta .cta-text {
+                   margin:0;
+                                  }
+  .main-content .middle-img {
+       width: 100%;
+    }
+
+    .main-content {
+       margin: 75px 20px 35px;
+    }
+
+    .text-content {
+        margin: 0 10px;
+    }
+
+    .contact {
+        margin: 0 30px;
+    }
+                                
+}
+/* mobile*/
+@media (max-width:500px){
+    .container {
+    background:red;
+    
+    }
+  header{
+      justify-content: center;
+  }
+    header nav {
+        flex-direction: column;
+        margin: 0;
+        padding:0;
+        width: 100%;
+        
+        }
+        header nav a {
+            width: 100%;
+            display: block;
+            text-align: center;
+            padding: 20px 0;
+            font-size: 20px;
+            border-bottom: 1px solid gray;
+        }
+
+        header nav a:first-child {
+        border-top: 1px solid gray;
+    }
+
+    header nav a:hover {
+        background: gray;
+    }
+    header .logo {
+    margin: 20px 0;
+}
+.cta{
+    margin-top: 50px;
+        flex-flow: column;
+        align-items: center; 
+}
+.cta img{
```

Comment:
This block of code is unnecessary. The .cta img should will not display in mobile view because we used `display: none` in the tablet media query.